gcputil/credentials.go

```diff
@@ -172,24 +205,44 @@ func ServiceAccountPublicKey(serviceAccount string, keyId string) (interface{},
 	return PublicKey(kStr)
 }
 
-// OAuth2RSAPublicKey returns the PEM key file string for Google Oauth2 public cert for the given 'kid' id.
-func OAuth2RSAPublicKey(ctx context.Context, keyId string) (interface{}, error) {
```

Comment:
Should be okay on the function signature here in terms of API breakage. This context wasn't being used though, so that's now fixed.